python/hsfs/feature_view.py

```diff
@@ -1107,39 +1809,159 @@ def get_train_validation_test_split(
         return df
 
     def add_training_dataset_tag(self, training_dataset_version: int, name: str, value):
+        """Attach a tag to a training dataset.
+
+        !!! example 
+            ```python
+            # get feature store instance 
+            fs = ...
+
+            # get feature feature view instance 
+            feature_view = fs.get_feature_view(...)
+
+            # attach a tag to a training dataset
+            feature_view.add_training_dataset_tag(
+                training_dataset_version=1, 
+                name="tag_schema", 
+                value={"key", "value"}
+            )
+            ```
+        """
         return self._feature_view_engine.add_tag(
             self, name, value, training_dataset_version=training_dataset_version
         )
 
     def get_training_dataset_tag(self, training_dataset_version: int, name: str):
+        """
+        !!! example 
+            ```python
+            # get feature store instance 
+            fs = ...
+
+            # get feature view instance 
+            feature_view = fs.get_feature_view(...)
+
+            # get a training dataset tag
+            feature_view.get_training_dataset_tag(
+                training_dataset_version=1,
+                 name="tag_schema"
+            )
+            ```
+        """
         return self._feature_view_engine.get_tag(
             self, name, training_dataset_version=training_dataset_version
         )
 
     def get_training_dataset_tags(self, training_dataset_version: int):
+        """
+        !!! example 
+            ```python
+            # get feature store instance 
+            fs = ...
+
+            # get feature view instance 
+            feature_view = fs.get_feature_view(...)
+
+            # get a training dataset tags           
+            feature_view.get_training_dataset_tags(
+                training_dataset_version=1
+            )
+            ```
+        """
         return self._feature_view_engine.get_tags(
             self, training_dataset_version=training_dataset_version
         )
 
     def delete_training_dataset_tag(self, training_dataset_version: int, name: str):
+        """
+        !!! example 
+            ```python
+            # get feature store instance 
+            fs = ...
+
+            # get feature view instance 
+            feature_view = fs.get_feature_view(...)
+
+            # delete training dataset tag        
+            feature_view.delete_training_dataset_tag(
+                training_dataset_version=1,
+                name='name_of_dataset'
+            )
+            ```
+        """
         return self._feature_view_engine.delete_tag(
             self, name, training_dataset_version=training_dataset_version
         )
 
     def purge_training_data(self, version: int):
```

Comment:
Here we are calling it 'version'.
But it's "training_dataset_version" everywhere else, right?